coalahtml/_coalahtml/app/app.js

```diff
@@ -2,12 +2,35 @@
 
 angular
   .module('coalaHtmlApp', ['ngRoute', 'ngSanitize'])
-  .run(function($rootScope) {
+  .run(function($rootScope, $http) {
```

Comment:
I'm not the Angular expert, but I think it may be better to inject arguments with list notation. This way you'll be able to minify your js files without problems and it'll be easier to test with dependency injections. Please correct me, if I'm missing something.